platform/osx/joypad_osx.cpp

```diff
@@ -55,9 +55,10 @@ void joypad::free() {
 	if (device_ref) {
 		IOHIDDeviceUnscheduleFromRunLoop(device_ref, CFRunLoopGetCurrent(), GODOT_JOY_LOOP_RUN_MODE);
 	}
-	if (ff_device) {
+	if (ff_device != nullptr) {
```

Comment:
Pointers evaluate well as booleans so we usually use the shorter `if (pointer)` style instead of making the `!= nullptr` part explicit.

Reply:
I was following the style on line 247. I can change both though?

Reply:
Yeah it's not very important, but for consistency's sake I guess you can change the other two occurrences on lines 247 and 607.

Reply:
Done!